### PR TITLE
tools: acrnd: update README.rst

### PR DESCRIPTION
**Pull request by taoyuhong**
Reviewed-by: Yan Like <like.yan@intel.com>
Reviewed-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>
Signed-off-by: Tao Yuhong <yuhong.tao@intel.com>

### CODE REVIEW

**Comment by NanlinXie** [comment]
Can one of the admins verify this patch?

File: tools/acrn-manager/README.rst [hunk 1]
```diff
@@ -93,10 +93,38 @@ Use the ``stop`` command to stop one or more running VM:
...
+.. _acrnd:
+
+acrnd
+#######
+
+
+Description
+***********
+
+The ``acrnd`` is the daemon for acrn-manager.
```
Comment (dbkinder) [inline tools/acrn-manager/README.rst]: Some grammar and clarity edits.  You should verify I've got the technical details right :)

    The ``acrnd`` daemon process provides a way for launching
    or resuming a UOS should the UOS shut down, either planned or
    unexpected. A UOS can ask ``acrnd`` to set up a timer to make
    sure the UOS is running, even if the SOS is suspended or stopped.
    The ``acrnd`` daemon stores pending UOS work to ``/opt/acrn/conf/timer_list``
    and sets an RTC timer to wake up the SOS or bring the SOS back up again.
    When ``acrnd`` daemon is restarted, it restores the previously saved timer
    list and launches the UOSs at the right time.

Comment (taoyuhong) [reply tools/acrn-manager/README.rst]: I believe it is good, I would like to use it.

**Comment by dbkinder** [comment]
CI build failures seem unrelated to this documentation change, so I'll merge this.

**Pull request merged**